packages/neo-one-client-core/src/__data__/transactions.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import {
+  ASSET_TYPE,
+  RegisterTransaction,
+  ScriptBuilder,
+  common,
+  crypto,
+} from '../';
```

Comment:
Move this to `neo-one-client` and change this line to `} from '@neo-one-client-core'`